lib/Header/Header.less

```diff
@@ -116,6 +116,62 @@
 	.moon-header-client-text {
 		line-height: @moon-button-small-height;
 	}
+
+	//InputHeader classes
+	&.moon-input-header {
+		.moon-input-header-input-decorator {
+			margin: -1px 0px 0px;
+			padding: 0px;
+			border: 0px;
+			width: 100%;
+			box-sizing: border-box;
+			background-color: transparent;
+
+			> .moon-input {
+				.moon-header-text;
+				margin: 0px;
+				padding-left: 1px;
+				padding-right: 1px;
+				display: inline-block;
+				box-sizing: border-box;
+				line-height: 1em;
+				color: @moon-input-header-text-color;
+				width: 100%;
+				text-overflow: ellipsis;
+			}
+
+			&.spotlight {
+				.moon-input {
```

Comment:
You'll have to make sure that the pseudo-selectors inherit the _right_ color, but otherwise it should be dandy.